components/browser/engine-gecko/src/test/java/mozilla/components/browser/engine/gecko/GeckoEngineSessionTest.kt

```diff
@@ -1127,7 +1127,7 @@ class GeckoEngineSessionTest {
     }
 
     @Test
-    fun `WHEN updating tracking protection with a recommended policy THEN etpEnabled should be enabled`() {
+    fun `WHEN updateing tracking protection with a recommended policy THEN etpEnabled should be enabled`() {
```

Comment:
typo

Reply:
Oh. This code was just moved over. This means this typo must be in the beta and probably nightly component too.

Reply:
Not blocking.